### PR TITLE
Data Challenge 0 Updates

### PR DESCRIPTION
**Pull request by smsimon**
Updates to the instrument model for DC0 runs.

### CODE REVIEW

File: s4sim/hardware/config.py [hunk 1]
```diff
@@ -682,6 +682,19 @@ def get_example():
         "SPLAT_HF": ["SPLAT_f220", "SPLAT_f280"],
         "SAT_HF": ["SAT_f220", "SAT_f280"],
     }
+    pins = {
```
Comment (keskitalo) [inline s4sim/hardware/config.py:685]: What are these? Perhaps adding a comment line would clarify it.

File: s4sim/hardware/config.py [hunk 2]
```diff
@@ -1641,67 +1651,46 @@ def get_example():
...
-    tele["tubespace"] =  174.3
+    tele["platescale"] = 0.00429
+    # This tube spacing in mm corresponds to 0.94 degrees projected on
+    # the sky at a plate scale of 0.00429 deg/mm
+    # The physical tube spacing may be different.
+    tele["tubespace"] =  0.94/0.00429
     tele["fwhm"] = lfwhm
     telescopes["LAT2"] = tele
 
     #from the DSR
```
Comment (keskitalo) [inline s4sim/hardware/config.py:1662]: This comment is probably stale

File: s4sim/hardware/config.py [hunk 3]
```diff
@@ -1641,67 +1651,46 @@ def get_example():
...
     sfwhm = OrderedDict()
-    sfwhm["SAT_f030"] = 72.8
-    sfwhm["SAT_f040"] = 72.8
-    sfwhm["SAT_f085"] = 25.5
-    sfwhm["SAT_f145"] = 25.5
-    sfwhm["SAT_f095"] = 22.7
-    sfwhm["SAT_f155"] = 22.7
-    sfwhm["SAT_f220"] = 13.0
-    sfwhm["SAT_f280"] = 13.0
+    sfwhm["SAT_f030"] = 85.2
```
Comment (keskitalo) [inline s4sim/hardware/config.py:1664]: Some significant changes here... We will have to double check that @zonca  has used the appropriate FWHM to generate the beam-smoothed skies.

**Review by keskitalo — APPROVED** [review]
Looks great. I would add one more comment line and change another.

File: s4sim/hardware/config.py (hunk 3 above)
Comment (zonca) [reply s4sim/hardware/config.py:1664]: I used https://github.com/CMB-S4/s4mapbasedsims/blob/master/202102_design_tool_run/instrument_model/cmbs4_instrument_model.tbl

File: s4sim/hardware/config.py (hunk 2 above)
Comment (smsimon) [reply s4sim/hardware/config.py:1662]: This is actually up to date with the new numbers for SPLAT

**Comment by jdborrill** [comment]
That's almost certainly wrong, and if it's not wrong it is horribly
mis-named.

We should be using the just-frozen PBD instrument model from Sara.

Julian

File: s4sim/hardware/config.py (hunk 3 above)
Comment (keskitalo) [reply s4sim/hardware/config.py:1664]: @zonca I'm afraid the numbers in your table match the old ones. We will need to update the SPLAT and SPSAT simulations.

File: s4sim/hardware/config.py (hunk 1 above)
Comment (smsimon) [reply s4sim/hardware/config.py:686]: Added a comment

File: s4sim/hardware/config.py (hunk 3 above)
Comment (smsimon) [reply s4sim/hardware/config.py:1665]: Yes, these changed signifcantly

**Comment by tskisner** [comment]
The changes in this `DC0` branch were implemented in #30, checked by @smsimon , and are now merged into master.  I suggest we delete this PR and open a new PR against master to address any lingering issues.

**Comment by smsimon** [comment]
Agreed

**Pull request closed without merging**